tests/test_fco_metrics.py

```diff
@@ -58,94 +65,107 @@ def test_fco_swu():
     if not HAVE_PYNE:
         raise SkipTest
     exp = pd.DataFrame(np.array([(0, 0.002407), (1, 0.001473)],
-        dtype=ensure_dt_bytes([('Year', '<i8'), ('SWU', '<f8')]))
-        )
+                                dtype=ensure_dt_bytes([('Year', '<i8'), ('SWU', '<f8')]))
+                       )
     mats = pd.DataFrame(np.array([
-        (UUID('f22f2281-2464-420a-8325-37320fd418f8'), 5, 7, 3, 3, 922350000, 8.328354),
-        (UUID('f22f2281-2464-420a-8325-37320fd418f8'), 5, 7, 3, 3, 922380000, 325.004979),
-        (UUID('f22f2281-2464-420a-8325-37320fd418f8'), 6, 8, 4, 3, 922350000, 11.104472),
-        (UUID('f22f2281-2464-420a-8325-37320fd418f8'), 6, 8, 4, 3, 922380000, 322.228861),
-        (UUID('f22f2281-2464-420a-8325-37320fd418f8'), 7, 9, 5, 12, 922350000, 11.104472),
-        (UUID('f22f2281-2464-420a-8325-37320fd418f8'), 7, 9, 5, 12, 922380000, 322.228861),
-        ], dtype=ensure_dt_bytes([
-                ('SimId', 'O'), ('QualId', '<i8'), ('ResourceId', '<i8'),
-                ('ObjId', '<i8'), ('TimeCreated', '<i8'), ('NucId', '<i8'),
-                ('Mass', '<f8')]))
-        )
+        (UUID('f22f2281-2464-420a-8325-37320fd418f8'),
+         5, 7, 3, 3, 922350000, 8.328354),
```

Comment:
Sometimes I don't like PEP8 :) - (not requesting a change - just commenting...)

Reply:
I do agree in this case,